src/renderer/components/GroupBase.js

```diff
@@ -66,6 +88,10 @@ class GroupBase extends React.Component {
         </Navbar>
         <div className='window'>
           <div className='GroupBase'>
+            <div className='SelectGroupImage'>
+              <img className='GroupImage' src={image} onClick={this.onSelectGroupImage.bind(this)} />
```

Comment:
yeah i wonder what this resolves to in the build if we used `path.join` on windows, or if it would just be fine.